tests/integration-service/namespace-backed-environments.go

```diff
@@ -123,10 +125,10 @@ var _ = framework.IntegrationServiceSuiteDescribe("Namespace-backed Environment
 		It("checks for deploymentTargetClaim after Ephemeral env has been created", func() {
 			Eventually(func() error {
 				dtcl, err = f.AsKubeDeveloper.GitOpsController.GetDeploymentTargetClaimsList(testNamespace)
-				if dtcl.Items == nil {
-					return fmt.Errorf("DeploymentTargetClaim is nil.")
+				if dtcl.Items == nil || len(dtcl.Items) == 0 {
+					return fmt.Errorf("No DeploymentTargetClaim is found.")
 				}
-				if dtcl.Items[0].Status.Phase != phaseDTC {
+				if !reflect.ValueOf(dtcl.Items[0].Status).IsZero() && dtcl.Items[0].Status.Phase != phaseDTC {
 					return fmt.Errorf("DeploymentTargetClaimPhase is: " + string(dtcl.Items[0].Status.Phase) + " Expected: " + string(phaseDTC))
```

Comment:
In case of non-existent "Status" field, this line could throw a NPE, so we can keep this error generic, something like:
```suggestion
					return fmt.Errorf("DeploymentTargetClaimPhase is not yet equal to the expected phase: " + string(phaseDTC))
```